packages/algob/src/lib/deployer.ts

```diff
@@ -20,12 +21,12 @@ export class AlgobDeployerImpl implements AlgobDeployer {
   private readonly algoClient: AlgoDeployClient;
 
   constructor (
-    runtimeEnv: AlgobRuntimeEnv, cpData: CheckpointRepo, asaDefs: ASADefs, algoSdk: AlgoDeployClient
+    runtimeEnv: AlgobRuntimeEnv, cpData: CheckpointRepo, asaDefs: ASADefs, algoClient: AlgoDeployClient
```

Comment:
reanme `algoClient`. It makes confusion with a client for the node.

Why you didn't use Operator as suggested?

Reply:
Because I want to have all the methods in this class so that Deployer class would be dumb -- have mostly oneliner functions. 
And simply would offload logic to this one.

Reply:
Renaming to `algoActions` and `AlgoActions`, and `AlgoActionsImpl`

Reply:
but this is not related to the `algosdk.Action` type? correct?

Reply:
It is not related.